tools/create_basis_compressed_glbs.py

```diff
@@ -120,6 +120,11 @@ def _gltf2unlit(gltf_name: str):
     with open(gltf_name, "r") as f:
         json_data = json.load(f)
 
+    if "extensionsUsed" in json_data:
+        json_data["extensionsUsed"].append("KHR_materials_unlit")
+    else:
+        json_data["extensionsUsed"] = ["KHR_materials_unlit"]
```

Comment:
I wonder, should this be in `extensionsRequired` as well? I.e., forcing the users to *always* render as flat/unlit. Or does Habitat have an option to also render them as Phong?